pandas/core/generic.py

```diff
@@ -3455,7 +3455,7 @@ def replace(self, to_replace=None, value=None, inplace=False, limit=None,
         ----------
         method : {'linear', 'time', 'index', 'values', 'nearest', 'zero',
                   'slinear', 'quadratic', 'cubic', 'barycentric', 'krogh',
-                  'polynomial', 'spline' 'piecewise_polynomial', 'pchip',
+                  'polynomial', 'spline' 'from_derivatives', 'pchip',
```

Comment:
no all of this validation stuff is in ``core/missing.py``. all we need here is a doc note. e.g. versionadded 0.18.1 -> from_derivates (and how scipy < 0.18.0) works with this.